language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/signature/SignatureHelpUtil.java

```diff
@@ -190,8 +268,8 @@ private static SignatureInfoModel getSignatureInfoModel(FunctionSymbol functionS
             documentation.get().parameterMap().forEach(paramToDesc::put);
         }
         // Add parameters and rest params
-        functionSymbol.typeDescriptor().params().get()
-                .forEach(param -> parameters.add(new Parameter(param, false, false, context)));
+        parameters.addAll(functionSymbol.typeDescriptor().params().orElse(new ArrayList<>()).stream()
```

Comment:
Get the stream's result to a variable and then add. This would clean the code.

Reply:
Done